packages/clay-date-picker/src/index.tsx

```diff
@@ -150,7 +160,8 @@ interface IProps extends React.HTMLAttributes<HTMLInputElement> {
 	onExpandedChange?: TInternalStateOnChange<boolean>;
 }
 
-const dateNow = new Date();
+const DATE_NOW = new Date();
```

Comment:
I know this was like this before, but this seems bad. It means that "now" means the time the module was loaded and not when the component was rendered. That may sound a bit edge casey, but I would expect this to misbehave around midnight at the end of the month, and the end of the year. The cost of avoiding the bug looks basically zero to me (just move it inside the component), so may as well do it. What do you think?

Reply:
Yes, it also makes sense. So, Can I just use `new Date();` in every place that I use?

Reply:
> I would just rename the variable to something like `const NEW_DATE = new Date()`.

Yep, that's exactly what I was thinking when I said:

> just move it inside the component